src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { CardGrid } from '@edx/paragon';
+import { useParams } from 'react-router-dom';
+import ContentHighlightCardItem from './ContentHighlightCardItem';
+import { TEST_COURSE_HIGHLIHTS_DATA } from './data/constants';
+
+const ContentHighlightsCardItemsContainer = () => {
+  const { highlightUUID } = useParams();
+  const [highlightCourses] = useState(
+    TEST_COURSE_HIGHLIHTS_DATA.filter(highlight => highlight.uuid === highlightUUID)[0].courses,
+  );
+
+  return (
+    <>
+      {highlightCourses && (
```

Comment:
Instead of putting this conditional inline with the JSX, you might also handle this with in its own `return` case, e.g.:

```jsx
if (!highlightCourses) {
  return null;
}

return (
  <CardGrid />
);
```